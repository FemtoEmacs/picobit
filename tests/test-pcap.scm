;; test for libpcap
(network-init)
