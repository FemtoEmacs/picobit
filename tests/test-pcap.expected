:0080000080
:06800000FBD70000F1D8DF
:00000001FF
